metricbeat/module/haproxy/_meta/kibana/6/dashboard/Metricbeat-haproxy-backend.json

```diff
@@ -2,7 +2,7 @@
     "objects": [
         {
             "attributes": {
-                "description": "", 
+                "description": "Haproxy backend metrics",
```

Comment:
Should we call it `HAProxy`? Same below

Reply:
Yes, changed all appearances of HAProxy with different uppercasing.